test/application.js

```diff
@@ -27,7 +26,7 @@ describe('app', function(){
     request(app.listen())
     .get('/')
     .end(function(){});
-  })
+  });
 
   it('should not .writeHead when !socket.writable', function(done){
```

Comment:
Why isn't there a space before the `{` like in the diffs above (`/lib`)? Is it because this is part of a function call?